tests/data/manifest/branding-xenserver.json

```diff
@@ -1,5 +1,6 @@
 {
     "sources": {}, 
+    "schemaVersion": "1", 
```

Comment:
This is on purpose. If you read schemaVersion 1 it means that the other fields will be empty